tests/openvino/test_modeling.py

```diff
@@ -547,6 +549,45 @@ def test_load_from_hub_and_save_sam_model(self):
         del model
         gc.collect()
 
+    def test_load_from_hub_and_save_text_speech_model(self):
```

Comment:
adding missing test to check we keep compatibility with openvino model pushed on the hub